sidebars.js

```diff
@@ -38,6 +38,7 @@ const sidebars = {
         "articles/step-2-add-api-key-auth",
         "articles/step-3-add-rate-limiting",
         "articles/step-4-deploying-to-the-edge",
+        "articles/step-5-project-sharing",
```

Comment:
Let me know if there's a better place than the `Getting Started` section for this.

Reply:
I suppose you could just add it to the `How To` Guides. @ntotten any preference?